tracking/anonymous_tracking.py

```diff
@@ -15,29 +18,35 @@
 logging.getLogger('posthog').disabled = True
 
 
+@dataclass
+class AnonymousWarehouse:
```

Comment:
Let's use pydantic.
It is better to start and handle schemas in one way, and pydantic is much stronger and will better in the long run.

Reply:
Plus you can move the hash method for the warehouse id into the schema.